src/main/java/nl/tudelft/scrumbledore/GUI.java

```diff
@@ -366,6 +372,9 @@ private void advanceLevel() {
    */
   private void checkPlayerAlive() {
     if (!game.getCurrentLevel().getPlayer().isAlive()) {
+      // Logging that the game has been restarted.
+      Logger.log("--------------------PLAYER RESTARTED THE GAME--------------------");
```

Comment:
Why not just leave the "-" signs and let the sentence start at the beginning of the line?